first_year_scraper/main.py

```diff
@@ -0,0 +1,18 @@
+import csv
+import excel_formatter
+import restructure
+
+filename = "wb1.xlsx"
```

Comment:
How about changing this to accept a CLI argument? We would be able to run the script directly, instead of having to change the code :sweat_smile: 

```sh
$ python main.py <path-to-wb1.xlsx>
```

Reply:
Yes! I'll do that, that would make the script really easy to use.

Reply:
@iakshat Are we doing this? If I understand correctly, @icyflame meant that we should add argument parsing for the `pdf` file that we are asking the user to input by prompting. Do you think this would be useful to do?

Ref - https://docs.python.org/3/library/argparse.html

Reply:
@thealphadollar  Ohh I think I misunderstood the comment of @icyflame to just taking filename as input. But I think as a user perspective this is easier for him/her to execute even with less/no knowledge of the script contents. If you are willing to give me some of your perspectives for the arg parsing, I might find it better to shift to that idea.

Reply:
Yeah, I think it's cool to ask the user. It is much easier and more friendly, I agree. 

However, @icyflame is the one who will resolve / accept your suggestion as this is a change suggested originally by him, and he has more experience to understand these matters :heart:

Reply:
I'm resolving this as it's been a long time since the last response. @icyflame Please reopen if you have any comments.